### ISSUE TITLE
wsltty does not work if user profile folder has a space in it

### ISSUE THREAD

**Issue by ryancdotorg** [mintty/wsltty#105]
My user profile folder is `C:\Users\Ryan Castellucci`

When I start wsltty, I very briefly see a console window showing

```
wslbridge error: failed to start backend process
note: backend error output: /bin/bash: /mnt/c/Users/Ryan: No such file or directory
```

**Comment by Biswa96** [comment]
Duplicate of #49. See last three comments. Solved in wslbridge.

**Comment by mintty** [comment]
The fixed version is not part of the current wsltty release yet, however.
The next release is in planning, it can only be a matter of weeks :/ (or maybe sooner).

**Comment by mintty** [comment]
Released 1.9.0 with updated wslbridge which should fix this.
